### PR TITLE
Fuse consecutive clamps into a single clamp (#21013)

### PR DESCRIPTION
**Pull request by apullin**
Summary:

ConvertToClampPass normalizes hardtanh, relu and relu6 into aten.clamp. Chains like HardTanh -> ReLU therefore become two adjacent clamps, the second of which is redundant.

Add FuseConsecutiveClampsPass, which folds any chain of adjacent (single-user) clamp.default ops into one clamp using exact composition:
`clamp(clamp(x,a,b),c,d) == clamp(x, max(a,c), min(b,d))`

This subsumes `HardTanh->ReLU`, `ReLU->ReLU`, `HardTanh->HardTanh`, `ReLU6->ReLU`, explicit clamp->clamp, and longer chains, dropping one op (and one intermediate tensor) per fused pair.

Runs after `FoldAndAnnotateQParamsPass` so it fires in both the FP and INT paths; in the quantized case the surviving clamp inherits the first clamp's input qparams and the second's output qparams, dropping the intermediate requant (qtol=1, same tradeoff as `FuseConsecutiveRescalesPass`).

Branching clamps (multiple users) are left untouched.

Reviewed By: rascani

Differential Revision: D112351522

### CODE REVIEW

**Comment by rascani** [comment]
Can you fix up the lintrunner? Thanks!

**Comment by apullin** [comment]
@rascani  Should be good to go, now.

File: backends/arm/_passes/fuse_consecutive_clamps_pass.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Copyright 2026 Arm Limited and/or its affiliates.
```
Comment (rascani) [inline backends/arm/_passes/fuse_consecutive_clamps_pass.py]: Should be Meta copyright since author is from Meta. Arm will add theirs if/when they edit the file.

**Review by rascani — APPROVED** [review]

**Comment by apullin** [comment]
@rascani copyright line resolved. Updates are to get lint OK again. Also added one test to exercise empty-range guard, per AI reviewer.
